phpcs.xml.dist

```diff
@@ -6,24 +6,15 @@
     <arg name="cache" value=".phpcs-cache"/>
     <arg name="colors"/>
 
-    <config name="php_version" value="70100"/>
+    <config name="php_version" value="80100"/>
 
     <!-- Ignore warnings, show progress of the run and show sniff names -->
     <arg value="nps"/>
 
     <file>lib</file>
     <file>tests</file>
 
-    <rule ref="Doctrine">
-        <exclude name="SlevomatCodingStandard.TypeHints.DeclareStrictTypes"/>
-        <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint"/>
-        <exclude name="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint"/>
-
-        <!-- Since Collections are built around mixed content, this has no value here -->
```

Comment:
That statement might be outdated: removing this block did not force me to add extra type declarations, probably thanks to the existing generic annotations.